packages/webapp/src/components/forms/AddClientForm/index.tsx

```diff
@@ -53,6 +55,11 @@ export const AddClientForm: StandardFC<AddClientFormProps> = wrap(function AddCl
 	const [submitMessage, setSubmitMessage] = useState<string | null>(null)
 	const [isSubmitButtonDisabled, setSubmitButtonDisabledState] = useState<boolean>(false)
 
+	// Extract first and last name from the name prop by taking everything before the first space as the first name
+	// and everything after as the last name
+	const [firstName, ...restOfName] = name.split(' ').map((namePart) => namePart.trim())
+	const lastName = restOfName.join(' ')
```

Comment:
Smart!
I wished the app was better and only had one field `name` specially with all the Spanish names they encounter.

Reply:
One issue is, if the user searches by last name the that will be put in the first name field, which will be annoying for the user. I figure we can get some feedback on this later.